v03_pipeline/lib/reference_datasets/gnomad.py

```diff
@@ -0,0 +1,58 @@
+import hail as hl
+
+from v03_pipeline.lib.model import ReferenceGenome
+
+
+def global_idx_field(reference_genome: ReferenceGenome) -> str:
+    return 'gnomad' if reference_genome == ReferenceGenome.GRCh37 else 'adj'
+
+
+def af_popmax_expression(
+    ht: hl.Table,
+    reference_genome: ReferenceGenome,
+) -> hl.Expression:
+    if reference_genome == ReferenceGenome.GRCh37:
+        return ht.popmax[ht.globals.popmax_index_dict['gnomad']].AF
+    return ht.grpmax['gnomad'].AF if hasattr(ht.grpmax, 'gnomad') else ht.grpmax.AF
```

Comment:
`hasattr(ht.grpmax, 'gnomad')` seems like something that should either always be true or false for a given file so shouldn't we just use whichever one is true for the latest release?

Reply:
I could use`hl.coalesce` here instead

Reply:
Its more that `hasattr` should always be the same for every row - for a given row I value may be missing or not but structs always need to have the same columns defined

Reply:
Is it okay to run `hasattr(ht.grpmax, 'gnomad')` on every row though? should I run `hasattr(ht.grpmax, 'gnomad')` before the `ht.select` instead?

Reply:
I think my issue is more that theres one table we use and its hardcoded so we should not be running conditional code as part of this loading process, the loading should be deterministic.  I think you should run that command locally, determine the structure of the table we are using, and then have this function use the fields that exist and not maintain code for fields that do not

Reply:
So 38 genomes has the nested `gnomad` struct inside `grpmax` and 38 exomes does not, and that is why this conditional is here. It seems to be the only difference between loading for gnomad exomes and genomes. 

I made new `gnomad_genomes` and `gnomad_exomes` modules that call a shared `get_ht` from a `gnomad_utils` module and provide a `af_popmax_expression` function as an argument. I also got rid of the `MODULE_NAME` config attribute.